.circleci/config.yml

```diff
@@ -18,13 +18,25 @@ var_2: &install-dependencies
       # install git and bash
       nix-env -f '<nixpkgs>' -i -A gitAndTools.git -A bash
 
+var_3: &install-cachix
+  run:
+    name: Install Cachix
+    shell: /usr/bin/env bash -euxo pipefail
+    command: |
+      if [[ -r "${HOME}/.nix-profile/etc/profile.d/nix.sh" ]]; then
+        source "${HOME}/.nix-profile/etc/profile.d/nix.sh"
+      fi
+
       # install cachix from nixpkgs at stable
       readonly shabka_path="$( pwd )"
-      release="$( tr -d "\n" < "${shabka_path}/.release" )"
-      export NIX_PATH="$( "${shabka_path}/lib/bash/nix-path.sh" "${release}" )"
+      release="$( tr -d "\n" < "${shabka_path}/release" )"
+
+      source "${shabka_path}/bin/shabka"
```

Comment:
```suggestion
```

Why are you sourcing the shabka binary?

Reply:
I need it here: https://github.com/kalbasit/shabka/pull/294/files#diff-1d37e48f9ceff6d8030570cd36286a61R34

Reply:
Fair enough!